### PR TITLE
Regenerated base code and all tests passed

### PR DESCRIPTION
**Pull request by mssfang**
- Regenerate code based on v3.1-preview.1 swagger and add service API version: v3.1-preview.1 as the latest version.
- Update internal implementation in order to pass all existing test.

fixes: #12980

### CODE REVIEW

File: sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/AnalyzeSentimentAsyncClient.java [hunk 1]
```diff
@@ -194,11 +195,13 @@ private AnalyzeSentimentResult convertToAnalyzeSentimentResult(DocumentSentiment
...
     private Mono<Response<AnalyzeSentimentResultCollection>> getAnalyzedSentimentResponse(
         Iterable<TextDocumentInput> documents, TextAnalyticsRequestOptions options, Context context) {
+        // TODO: add opinion mining in the following PR
         return service.sentimentWithResponseAsync(
             new MultiLanguageBatchInput().setDocuments(toMultiLanguageInput(documents)),
-            context.addData(AZ_TRACING_NAMESPACE_KEY, COGNITIVE_TRACING_NAMESPACE_VALUE),
             options == null ? null : options.getModelVersion(),
-            options == null ? null : options.isIncludeStatistics())
+            options == null ? null : options.isIncludeStatistics(),
+            null,
```
Comment (mssfang) [inline sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/AnalyzeSentimentAsyncClient.java:203]: adding null here. But will having opinion mining feature when doing this issue: #12982

File: sdk/textanalytics/azure-ai-textanalytics/src/test/resources/session-records/detectLanguagesBatchStringList.json [hunk 2]
```diff
@@ -1,27 +0,0 @@
-{
```
Comment (samvaity) [inline sdk/textanalytics/azure-ai-textanalytics/src/test/resources/session-records/detectLanguagesBatchStringList.json:1]: Were these unused test files?

Comment (mssfang) [reply sdk/textanalytics/azure-ai-textanalytics/src/test/resources/session-records/detectLanguagesBatchStringList.json:1]: yes. these are unused test files

File: sdk/textanalytics/azure-ai-textanalytics/src/test/resources/session-records/analyzeSentimentForEmptyText.json [hunk 3]
```diff
@@ -1,25 +1,4 @@
...
-      "apim-request-id" : "efbe390c-f5c2-41a3-886b-d6dcc12c2ab5",
-      "retry-after" : "0",
-      "StatusCode" : "200",
-      "Body" : "{\"documents\":[],\"errors\":[{\"id\":\"0\",\"error\":{\"code\":\"InvalidArgument\",\"message\":\"Invalid document in request.\",\"innererror\":{\"code\":\"InvalidDocument\",\"message\":\"Document text is empty.\"}}}],\"modelVersion\":\"2019-10-01\"}",
-      "Date" : "Wed, 29 Apr 2020 17:12:19 GMT",
-      "Content-Type" : "application/json; charset=utf-8"
-    },
-    "Exception" : null
-  } ],
+  "networkCallRecords" : [ ],
```
[the remark this answers is not in the document]
Comment (mssfang) [reply sdk/textanalytics/azure-ai-textanalytics/src/test/resources/session-records/analyzeSentimentForEmptyText.json]: you are looked at the older commit.

**Pull request merged**